### PR TITLE
Create base Progress component

### PR DESCRIPTION
**Pull request by zachstence**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +23,7 @@
     "@storybook/addon-postcss": "^2.0.0",
     "@storybook/addon-svelte-csf": "^1.1.0",
     "@storybook/svelte": "^6.3.2",
-    "@tsconfig/svelte": "^2.0.1",
+    "@types/faker": "^5.5.7",
```
Comment (0c370t) [inline package.json:26]: We do we have faker types installed if we don't have faker installed?

File: src/atoms/Progress/Progress.spec.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+
+
```
Comment (0c370t) [inline src/atoms/Progress/Progress.spec.ts:2]: Empty lines at start of file

File: src/atoms/Progress/Progress.spec.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        const {getByRole} = render(Progress, {
+            progress: 0,
+        });
+        const progress = getByRole("progressbar");
+        expect(progress).toHaveStyle({ "width": "0%" });
+    });
+
+    it("should show an empty bar when progress is below 0", () => {
+        const {getByRole} = render(Progress, {
+            progress: -9999,
```
Comment (0c370t) [inline src/atoms/Progress/Progress.spec.ts:19]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.spec.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+        });
+        const progress = getByRole("progressbar");
+        expect(progress).toHaveStyle({ "width": "0%" });
+    });
+
+    it("should show a full bar when progress=max", () => {
+        const {getByRole} = render(Progress, {
+            progress: 10,
+            max: 10
```
Comment (0c370t) [inline src/atoms/Progress/Progress.spec.ts:28]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.spec.ts [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        });
+        const progress = getByRole("progressbar");
+        expect(progress).toHaveStyle({ "width": "100%" });
+    });
+
+    it("should show a full bar when progress>max", () => {
+        const {getByRole} = render(Progress, {
+            progress: 11,
+            max: 10
```
Comment (0c370t) [inline src/atoms/Progress/Progress.spec.ts:37]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.spec.ts [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+        });
+        const progress = getByRole("progressbar");
+        expect(progress).toHaveStyle({ "width": "100%" });
+    });
+
+    it("should show a bar with size progress/max", () => {
+        const {getByRole} = render(Progress, {
+            progress: 10,
+            max: 100
```
Comment (0c370t) [inline src/atoms/Progress/Progress.spec.ts:46]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.stories.svelte [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+            control: "select",
+            options: [
+                "small",
+                "medium",
+                "large",
+                "fill"
+            ]
+        },
+        max: {
+            defaultValue: 20,
```
Comment (0c370t) [inline src/atoms/Progress/Progress.stories.svelte:21]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.stories.svelte [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+            control: {
+                type: "number",
+            }
+        },
+        progress: {
+            defaultValue: 13,
```
Comment (0c370t) [inline src/atoms/Progress/Progress.stories.svelte:27]: No Magic Numbers > 2

File: src/atoms/Progress/Progress.svelte [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    export type ProgressVariant = "small" | "medium" | "large" | "fill";
+</script>
+
+<script lang="ts">
+    export let progress: number;
+    export let max: number = 1;
+    export let variant: ProgressVariant = "medium";
+
+    $: actualProgress = progress < 0 ? 0 : progress > max ? max : progress;
+    $: width = (actualProgress / max) * 100;
```
Comment (0c370t) [inline src/atoms/Progress/Progress.svelte:11]: No Magic Numbers > 2

**Review by 0c370t — CHANGES_REQUESTED** [review]
Lots of magic numbers, needs an eslint pass

File: src/atoms/Progress/Progress.spec.ts (hunk 2 above)
Comment (zachstence) [reply src/atoms/Progress/Progress.spec.ts]: Fixed in 0f469ad

File: package.json (hunk 1 above)
Comment (zachstence) [reply package.json]: Forgot to uninstall, fixed in 0f469ad

**Review by 0c370t — APPROVED** [review]

**Pull request merged**